resources/content_list.ts

```diff
@@ -6,7 +6,7 @@ import ZoneId from './zone_id';
 // These are not things that cactbot necessarily supports, but things that it
 // theoretically could be supporting in the future with raidboss and oopsy.
 
-export default [
+const contentList: (number | null)[] = [
```

Comment:
Should ZoneId.MatchAll return like -1 instead so we don't have to check null?

Reply:
It could, but I'd want to change that in a different PR.  I can leave a TODO if you think that'd be better.

Reply:
Up to you; I don't think it's necessary. We can get to it whenever, if ever